ui/apps/platform/src/Containers/Clusters/ClustersSidePanel.tsx

```diff
@@ -107,13 +89,25 @@ function ClustersSidePanel({ selectedClusterId, setSelectedClusterId }) {
             const clusterIdToRetrieve = selectedClusterId;
 
             setLoadingCounter((prev) => prev + 1);
-            fetchCentralEnv()
-                .then((freshCentralEnv) => {
-                    setCentralEnv(freshCentralEnv);
+            getClusterDefaults()
+                .then((clusterDefaults) => {
+                    const {
+                        mainImageRepository: mainImage,
+                        collectorImageRepository: collectorImage,
+                        kernelSupportAvailable,
+                    } = clusterDefaults;
+
+                    setCentralEnv({
+                        kernelSupportAvailable,
+                        successfullyFetched: true,
+                    });
+
                     if (clusterIdToRetrieve === 'new') {
                         const updatedCluster = {
                             ...selectedCluster,
-                            slimCollector: freshCentralEnv.kernelSupportAvailable,
+                            mainImage,
+                            collectorImage,
+                            slimCollector: kernelSupportAvailable,
```

Comment:
KernelSupportAvailable means that kernel modules are available for live download by collector and collector can be started in "slim" mode, i.e. small container image without kernel modules pre-bundled.
It is the decision like "since kernel support is available, we'll create the cluster with slim collector" happening here. I guess, that's what it says. This decision needs to happen somewhere. It could have been pushed to backend, but it could also be presented differently on UI. Something like a popup dialog "We detected that your Central can download collector kernel drivers or has collector support package deployed. Would you like to deploy slim collector to your Secured Cluster? [Yes / No]"